data/crac-file/crac-impl/src/test/java/com/farao_community/farao/data/crac_impl/remedial_action/range_action/PstWithRangeTest.java

```diff
@@ -157,4 +158,58 @@ public void pstWithRelativeFixedRange() {
         assertEquals(phaseTapChanger.getStep(initialTapPosition - 3).getAlpha(), pst.getMinValue(network), 0);
         assertEquals(phaseTapChanger.getStep(initialTapPosition + 3).getAlpha(), pst.getMaxValue(network), 0);
     }
+
+    @Test
+    public void desynchronize() {
+        pst.addRange(new Range(-3, 3, RangeType.RELATIVE_FIXED, RangeDefinition.CENTERED_ON_ZERO));
+        pst.synchronize(network);
+        pst.desynchronize();
```

Comment:
Maybe you can check the value of pst.getMaxValue(network) between the synchronization and desynchronization, to make sure that the desynchronization works (and it is not just that the synchronization haven't done anything...)

Reply:
added